MarsCompetition/MarsCompetition/Utilities/JsonHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsCompetition.Utilities
+{
+    public class Jsonhelper
+    {
+        public static List<T> ReadTestDataFromJson<T>(string jsonFile)
+        {
+            string sCurrentDirectory = AppDomain.CurrentDomain.BaseDirectory;
+            //Console.WriteLine("From JSON Helper.. Current Directory: " +sCurrentDirectory);
```

Comment:
remove all commented out code it is bad practice to have them in code as it creates unwanted noise